techniques/system/inventory/1.0/fusionAgent.st

```diff
@@ -486,6 +486,9 @@ bundle agent addInformationsToInventory
     solaris::
       "RUDDERUUID" string => execresult("smbios | grep UUID | sed 's/  UUID: //'", "useshell");
 
+    freebsd::
+      "RUDDERUUID" string => execresult("/usr/local/sbin/dmidecode -s system-uuid","noshell");
```

Comment:
You should define RUDDERUUID_cmd instead since the execresult is done a few lines below.